charts/victoria-metrics-gateway/README.md

```diff
@@ -8,12 +8,15 @@ Victoria Metrics Gateway - Auth & Rate-Limitting proxy for Victoria Metrics
 
 # Table of Content
 
-* [Prerequisites](#prerequisites)
-* [Chart Details](#chart-details)
-* [How to Install](#how-to-install)
-* [How to Uninstall](#how-to-uninstall)
-* [How to use JWT signature verification](#how-to-use-jwt-signature-verification)
-* [Documentation of Helm Chart](#documentation-of-helm-chart)
+- [Victoria Metrics Helm Chart for vmgateway](#victoria-metrics-helm-chart-for-vmgateway)
+- [Table of Content](#table-of-content)
+  - [Prerequisites](#prerequisites)
+  - [Chart Details](#chart-details)
+  - [How to install](#how-to-install)
+  - [How to uninstall](#how-to-uninstall)
+- [How to use JWT signature verification](#how-to-use-jwt-signature-verification)
```

Comment:
Is there particular reason to change this index?

Reply:
No, you're right, that was my VS Code extension to auto-generate the table of contents on saving the file.
I reverted that change